### ISSUE TITLE
Remove the powerglove item

### ISSUE THREAD

**Issue by Makkkkus** [MinicraftPlus/minicraft-plus-revived#258]
**Is your feature request related to a problem? Please describe.**
The powerglove isn't really used anymore except in the back-end. The sprite has been removed, and as of now, there is not really any reason to keep it.

**Describe the solution you'd like**
Remove the power glove item from the items list, even if it is hidden.

**Describe alternatives you've considered**
Because we removed multiplayer, this shoudn't be any problem.

**Additional context**
Requires removing multiplayer code first.

**Comment by PseudoDistant** [comment]
I have no choice but to mod it back in then. :/
Also removing multiplayer conc

**Comment by BenCheung0422** [comment]
There is another possible solution apart from this (might be approached in mods) is that using power glove, picking up furniture does not cost stamina/energy, and in inverse without the use of power glove.

**Comment by Makkkkus** [comment]
Great example of feature creep. The power glove has already been removed, so this should probably ble closed.

**Comment by Makkkkus** [comment]
On second thought, maybe not, but there's nothing stopping us from removing it at this point. We're building a new multiplayer implementation, so a new system will be designed anyway.

**Comment by BenCheung0422** [comment]
Also, it is not completely removed as the class still exists.

**Comment by BenCheung0422** [comment]
I think this issue can be superseded by #567.

**Comment by zandgall** [comment]
Yes superceded by #567, but I do think it would be fine to get rid of power glove unless there really is something cool we can do with it
